kcards/models/room.py

```diff
@@ -43,38 +46,38 @@ def queue(self):
         items = []
 
         for name in self.red:
```

Comment:
Eventually, I think we'll want to rename the model lists as well to match the domain instead of just using the color.